vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
```

Comment:
`[Errno 95] Not supported: '/tmp/pip-u800vb8q-build/testing/resources/working_symlink'")]`
End there're many many other with code 95